__tests__/app.test.js

```diff
@@ -34,9 +34,58 @@ describe('api/topics', () => {
     })  
 })
 
+describe('api/articles', () => {
+    it('should respond with 200 if array of objects found', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((result) => {
+        expect(Array.isArray(result.body.articles)).toBe(true);  
+        expect(result.body.articles[0]).toHaveProperty('title');
+        expect(result.body.articles[0]).toHaveProperty('topic');
+        expect(result.body.articles[0]).toHaveProperty('author');
+        expect(result.body.articles[0]).toHaveProperty('body');
+        expect(result.body.articles[0]).toHaveProperty('created_at');
+        expect(result.body.articles[0]).toHaveProperty('votes');
+        expect(result.body.articles.length).toBe(12);
+        })
+    }) 
+})  
 
+describe('api/articles/:article_id', () => {
+    it('should respond with 200 if array of objects found', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then((result) => {
+            expect(result.body.article).toEqual({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: expect.any(String),
+                votes: 100
+              })
+            })
+        })
+    })
 
-
-
-
-
+    it('should generate 404 if items not found', () => {
+        return request(app)
+        .get('/api/articles/20')
+        .expect(404)
+        .then((result) => {
+        console.log(result.body, '<- result.body')
```

Comment:
Please remove this console log

Reply:
Thanks sorry I should have checked for these first!